src/app/donation-start/donation-start-container/donation-start-container.component.html

```diff
@@ -0,0 +1,41 @@
+<main class="b-container">
+  <div class="b-back-button">
+    <a mat-icon-button [routerLink]="'/campaign/' + campaign.id">
+      <mat-icon aria-hidden="false" aria-label="Back">keyboard_arrow_left</mat-icon>
+      Back to campaign
+    </a>
+  </div>
+  <div class="c-form-container">
+    <div class="b-primary-column">
+      <app-donation-start [campaign]="campaign" [column]="'primary'"/>
+    </div>
+    <div class="b-secondary-column">
+      <app-donation-start [campaign]="campaign" [column]="'secondary'"/>
```

Comment:
I couldn't find a way to define the two columns in the container component and then let the old component be split across two columns other than by including it twice as I've done here.